docs/data/pages.ts

```diff
@@ -140,9 +140,10 @@ const pages: MuiPage[] = [
           {
             pathname: '/x/react-data-grid/server-side-data-group',
             title: 'Server-side data',
+            unstable: true,
             plan: 'pro',
             children: [
-              { pathname: '/x/react-data-grid/server-side-data', title: 'Overview' },
+              { pathname: '/x/react-data-grid/server-side-data', title: 'Overview', plan: 'pro', },
```

Comment:
I would lobby to have this in the MIT plan, to make it easier to adopt the data grid against other options, and to make it easier to access Pro features, but ok, we can always easy change this.